client/src/utils/axios.js

```diff
@@ -0,0 +1,14 @@
+
+
+import axios from "axios";
+
+const apiClient = axios.create({
+  baseURL: 'http://localhost:8000/api/v1',
```

Comment:
put this in your .env file to allow dynamic routes, depending on user's environment (ex. I'm using port 8889 so I can't connect to your API because the base route is hard coded)